src/components/Missions.jsx

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable camelcase */
```

Comment:
- Hi, you have done an excellent Job. I would advise you not to disable the Linter. I am asking this because linter is a tool that analyzes your source code to flag programming errors, bugs, stylistic errors, and suspicious constructs(source: [Wikipedia](https://en.wikipedia.org/wiki/Lint_(software))). Working with Linter is more professional.